tests/test_salesforce_full_table_reset.py

```diff
@@ -0,0 +1,89 @@
+import unittest
```

Comment:
This appears to be a less rigorous FULL_TABLE test.  The concept of resetting a FULL_TABLE stream doesn't seem like a valid case since we do not expect to have a bookmark for a FULL_TABLE stream.  Since we do have FULL_TABLE streams for salesforce, id suggest replacing this test with a FULL_TABLE replication test.  There is an example of a SaaS full table test in the stripe test repo.

Reply:
Ok, Will create a separate card for full table replication and implement it. Will remove the full_table_reset from this PR.

Reply:
Removed the file